### DISCUSSION TITLE
Python的入门建议？

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#86] [灌水]
翔哥好，请教你入门python有什么好的教程推荐吗？我主要是想处理科研数据，加一些简单的机器学习，网上的资料太多了不知道什么比较好，因为现在时间不充裕做不到慢慢学筛选。之前学R的时候也是一顿乱看最后发现走了不少弯路，还是你写的教程比较好，但是python这块你写的少一些，看你的笔记应该也是比较精通python的，你觉得比较好的python学习资料有推荐的吗？感谢

**Comment by ShixiangWang** [comment]
通用的编程可以有 利用Python进行数据处理，Python数据科学手册等等。

专业性一点的分享下面的资料给你，之前收集的：

我用阿里云盘分享了「[Sebastian_Bassi]_Python...matics(b-ok.org).pdf」等多个文件，你可以不限速下载🚀
复制这段内容打开「阿里云盘」App 即可获取
链接：https://www.aliyundrive.com/s/UrnpWY4vFnH

**Reply by zhaoliang0302** [reply]
感谢推荐！
